lib/config.ts

```diff
@@ -58,6 +60,8 @@ export const getRataExtraStackConfig = (scope: Construct) => ({
   dmzApiEndpoint: getSSMStringParameter(scope, SSM_DMZ_API_DOMAIN_NAME),
   databaseDomain: getSSMStringParameter(scope, SSM_DATABASE_DOMAIN),
   jwtTokenIssuer: getSSMStringParameter(scope, SSM_JWT_TOKEN_ISSUER),
+  alfrescoAPIKeyName: getSSMStringParameter(scope, SSM_ALFRESCO_API_KEY_NAME),
+  alfrescoAPIUrlName: getSSMStringParameter(scope, SSM_ALFRESCO_API_URL_NAME),
```

Comment:
This will pass the actual value. Which is good! But as such, drop the Name from the variable name.